### DISCUSSION TITLE
TESS and PHOEBE Import

### DISCUSSION THREAD

**Discussion by jmcxabc** [phoebe-project/phoebe2#828] [Questions]
Hi, how do I import TESS data into PHOEBE for analysis? I am relatively new to PHOEBE on Colab and only really know how to edit PHOEBE's default binary. Please help!

**Comment by bpablo** [comment]
Hello,

We have a series of tutorials available on the Phoebe Project website. The answer to your particular question can be found here:

http://phoebe-project.org/docs/2.4/tutorials/datasets
